src/compositor/input/mod.rs

```diff
@@ -27,7 +27,12 @@ use smithay::{
         tablet_manager::{TabletDescriptor, TabletSeatTrait},
     },
 };
-use xkbcommon::xkb::{self, Keysym};
+use xkbcommon::xkb::{
+    self, Keysym,
+    keysyms::{
+        KEY_BackSpace, KEY_q, KEY_XF86Switch_VT_1, KEY_XF86Switch_VT_12, KEY_Return, KEY_1, KEY_9, KEY_M, KEY_P, KEY_W, KEY_R
+    }
```

Comment:
Hmmm.

I'd personally rather have it prefixed by some mod path, rather than putting all krys it all into the global namespace.

But I suppose they all start with `KEY_` anyway, so leave it.